js/src/contracts/githubhint.js

```diff
@@ -27,6 +28,19 @@ export default class GithubHint {
   }
 
   getInstance () {
-    return this.getContract().instance;
+    if (this._instance) {
+      return Promise.resolve(this._instance);
+    }
+
+    return this.getContract().then((contract) => {
+      this._instance = contract.instance;
+      return this._instance;
+    });
+  }
```

Comment:
Implementation aligns with what we have in the others in `~/contracts` now - previously it actually didn't work at all.